lint-tasks.yml

```diff
@@ -97,6 +139,38 @@ tasks:
           black --color --line-length 100 {{.BLACK_FLAGS}} .
           ruff check {{.RUFF_FLAGS}} .
 
+  linter-nodejs:
+    internal: true
+    vars:
+      CHECKSUM_FILE: "{{.BUILD_DIR}}/{{.TASK}}.md5"
+      OUTPUT_DIR: "{{.LINTER_NODEJS_BUILD_DIR}}"
+    cmds:
+      - task: ":nodejs"
+        vars:
+          NODEJS_VERSION: "latest"
+          OUTPUT_DIR: "{{.OUTPUT_DIR}}"
+      # Checksum the generated files (this command must be last)
+      - >-
+        tar cf - --directory "{{.OUTPUT_DIR}}" {{.CHECKSUM_TAR_BASE_ARGS}} .
+        | md5sum > "{{.CHECKSUM_FILE}}"
+    sources:
+      - "{{.TASKFILE_DIR}}/lint-tasks.yml"
+      - "{{.TASKFILE_DIR}}/Taskfile.yml"
+    status:
+      - "test -f '{{.CHECKSUM_FILE}}'"
+      - >-
+        diff
+        <(tar cf - --directory "{{.OUTPUT_DIR}}" {{.CHECKSUM_TAR_BASE_ARGS}} . | md5sum)
+        "{{.CHECKSUM_FILE}}"
+
+  node-modules:
+    internal: true
+    dir: "{{.TASKFILE_DIR}}/components/webui"
+    deps: ["linter-nodejs"]
+    cmds:
+      -  # `--no-package-lock`: prevent `npm` from checking the `package-lock.json` file version
+        "PATH='{{.LINTER_NODEJS_BIN_DIR}}':$PATH npm install --no-package-lock"
```

Comment:
* We can put the comment above the `-` rather than within it
* Since `--no-package-lock` doesn't seem to exist in newer versions (or at least it's not documented), perhaps we can just remove the package-lock.json file after installing?